6-AdvancedScenarios/4-hybrid-SPA/App/appSettings.js

```diff
@@ -0,0 +1,20 @@
+const appSettings = {
+    appCredentials: {
+        clientId: "Enter_the_Application_Id_Here",
+        tenantId: "Enter_the_Tenant_Info_Here",
+        clientSecret: "Enter_the_Client_Secret_Here",
```

Comment:
I would have these set by environment variables.

Reply:
@jasonnutter  Removed and replaced with .env file.